### PR TITLE
Revert "Revert "Perpetuals Migration #8 - Migrate main table (#4049)""

### PR DESCRIPTION
**Pull request by henrystats**
Reverts duneanalytics/spellbook#4111

### CODE REVIEW

**Comment by henrystats** [comment]
@jeff-dude I could only see the tag on the revert perpetual trades in the notifications box... Do I leave this pr as it is and just submit a new pr that revert the changes back to what they were originally merged as and then update the legacy file?

**Comment by jeff-dude** [comment]
> @jeff-dude I could only see the tag on the revert perpetual trades in the notifications box... Do I leave this pr as it is and just submit a new pr that revert the changes back to what they were originally merged as and then update the legacy file?

yeah, essentially reopen your exact PR that was just merged (not my revert) and then reset legacy back to what they were. you should be able to cherry pick certain commits, or if your branch is still active, even easier to just open a PR again

**Comment by jeff-dude** [comment]
> > @jeff-dude I could only see the tag on the revert perpetual trades in the notifications box... Do I leave this pr as it is and just submit a new pr that revert the changes back to what they were originally merged as and then update the legacy file?
> 
> yeah, essentially reopen your exact PR that was just merged (not my revert) and then reset legacy back to what they were. you should be able to cherry pick certain commits, or if your branch is still active, even easier to just open a PR again

sorry, just realized this is your new PR. this should be good.

**Comment by henrystats** [comment]
ohh!! amazing! oke thank you!

File: models/perpetual/perpetual_trades_legacy.sql [hunk 1]
```diff
@@ -1,9 +1,59 @@
...
-SELECT 
-1 as dummy 
\ No newline at end of file
+{% set perpetual_trade_models = [
+ ref('perpetual_protocol_perpetual_trades_legacy')
+,ref('pika_perpetual_trades_legacy')
+,ref('synthetix_perpetual_trades_legacy')
+,ref('emdx_avalanche_c_perpetual_trades_legacy')
+,ref('hubble_exchange_avalanche_c_perpetual_trades_legacy')
+,ref('gmx_perpetual_trades_legacy')
```
Comment (jeff-dude) [inline models/perpetual/perpetual_trades_legacy.sql:23]: if you aren't already actively looking, these would all need restored too

Comment (henrystats) [reply models/perpetual/perpetual_trades_legacy.sql:23]: sorry I don't understand what you mean by restored

Comment (henrystats) [reply models/perpetual/perpetual_trades_legacy.sql:23]: I didn't edit the legacy models of the projects, I originally did but then I reverted them, before they got reviewed/merged.

Comment (henrystats) [reply models/perpetual/perpetual_trades_legacy.sql:23]: only changes I made was to  perpetual_trades_legacy.sql

Comment (jeff-dude) [reply models/perpetual/perpetual_trades_legacy.sql:23]: ah, got it, sounds good. i will do final check and merge again in the morning 🤝

Comment (henrystats) [reply models/perpetual/perpetual_trades_legacy.sql:23]: thanks ser!

**Pull request merged**